.github/workflows/release-post-merge.yml

```diff
@@ -12,14 +12,36 @@ on:
       version:
         description: "Release version (Be sure `Release-branch` is successful):"
         required: true
-
+      image_repo:
+        type: choice
+        description: "Target image repository for built images"
+        default: mongodb/mongodb-atlas-kubernetes-operator-prerelease
+        required: true
+        options:
+        - mongodb/mongodb-atlas-kubernetes-operator-prerelease
+        - mongodb/mongodb-atlas-kubernetes-operator
+      release_helm:
```

Comment:
Once approved, should we cleanup those control flags? or at least set "prod" default values

Reply:
See env section on how default values get computed.